Pod/Classes/VSLEndpoint.m

```diff
@@ -207,6 +207,10 @@ - (BOOL)startEndpointWithEndpointConfiguration:(VSLEndpointConfiguration  * _Non
     mediaConfig.has_ioqueue = PJ_TRUE;
     mediaConfig.thread_cnt = 1;
     mediaConfig.no_vad = PJ_TRUE;
+    mediaConfig.jb_init = 2;
+    mediaConfig.jb_min_pre = 1;
+    mediaConfig.jb_max_pre = 3;
+    mediaConfig.jb_max = 4;
```

Comment:
these line should be removed, I should not have committed those. They were part of an unsuccessful experiment for https://voipgrid.atlassian.net/browse/VIALI-3649?oldIssueView=true . See the last comment: https://voipgrid.atlassian.net/browse/VIALI-3649?focusedCommentId=53689&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-53689

Reply:
done - removed